js/main.js

```diff
@@ -21,23 +22,91 @@ function getData() {
 }
 getData();
 
-var likedThought = document.querySelector('#form-input');
 var text;
 var authorName;
 
-function handleClick(event) {
+function handleLike(event) {
   event.preventDefault();
 
   var savedQuote = {
     quote: text,
-    author: authorName
+    author: authorName,
+    entryId: data.nextEntryId
   };
 
   data.nextEntryId++;
-  likedThought.reset();
-  data.entries.unshift(likedThought);
-  ul.prepend(savedQuote);
+  // likedThought.reset();
+  data.entries.unshift(savedQuote);
+  renderLiked(savedQuote);
+  viewSwap(savedQuote);
+}
+
+if ($button != null) {
+  $button.addEventListener('click', handleLike);
+}
+
+function renderLiked(entry) {
+  var card = document.createElement('li');
+  // card.className('card');
+  card.setAttribute('class', 'card');
+
+  var text = document.createElement('li');
+  text.textContent = entry.text;
+
+  var person = document.createElement('li');
+  person.textContent = entry.person;
+
+  ul.appendChild(card);
+  ul.appendChild(text);
+  ul.appendChild(person);
```

Comment:
You shouldn't append directly to the DOM within a DOM creation function, you should `return` the DOM tree so that you can use this same function to create the same format of a DOM tree for use anywhere in your document